### PR TITLE
implement the clean_agreement function (Issue #12)

### PR DESCRIPTION
**Pull request by Adegbem**
implement the clean_agreement function (Issue #12) one of the node in the data engineering phase

### CODE REVIEW

File: src/nlp_sdg/pipelines/data_engineering/pipeline.py [hunk 1]
```diff
@@ -5,19 +5,18 @@
...
 
 def create_pipeline(**kwargs) -> Pipeline:
     pipeline_instance =  pipeline(
         
         [
             node(
-                func=dummy_node,
+                func=clean_agreement,
                 inputs="sdg_data",
-                outputs="model_input_data",
```
Comment (Luka-Explore) [inline src/nlp_sdg/pipelines/data_engineering/pipeline.py:19]: @Adegbem There should be a new node() for the new function

**Comment by Nthiki** [comment]
@Adegbem you have branched off from **main** instead of **development**.

**Comment by Adegbem** [comment]
I Branch from development, I make sure of it, but I see @Luka-Explore as create another branch, maybe I can delete my branch and work with that branch

**Comment by Nthiki** [comment]
@Adegbem then it means that you've made a mistake when creating the PR, because it is trying to merge into the main branch instead of development
image

**Comment by Adegbem** [comment]
Yeah, i agree, the mistake is from the pull request

**Pull request closed without merging**